electrum/fee_policy.py

```diff
@@ -0,0 +1,369 @@
+        else:
+            fee_per_byte = fee_per_kb/1000
+            rate_str = format_fee_satoshis(fee_per_byte) + f" {util.UI_UNIT_NAME_FEERATE_SAT_PER_VBYTE}"
+
+        if self.use_dynamic_estimates:
+            tooltip = rate_str
+
+        elif self.method == FeeMethod.STATIC:
+            assert fee_per_kb is not None
+            assert fee_per_byte is not None
+            if network.mempool_fees.has_data():
+                depth = network.mempool_fees.fee_to_depth(fee_per_byte)
+                tooltip = self.depth_tooltip(depth)
+            else:
+                tooltip = ''
+            if network.fee_estimates.has_data():
+                eta = network.fee_estimates.fee_to_eta(fee_per_kb)
+                tooltip += '\n' + self.eta_tooltip(eta)
+        return tooltip
+
+    def get_tooltip(self, network: 'Network'):
+        target = self.get_target_text()
+        estimate = self.get_estimate_text(network)
+        if self.use_dynamic_estimates:
+            return _('Target') + ': ' + target + '\n' + _('Current rate') + ': ' + estimate
+        else:
+            return _('Fixed rate') + ': ' + target + '\n' + _('Estimate') + ': ' + estimate
+
+    @classmethod
+    def depth_tooltip(self, depth: Optional[int]) -> str:
+        """Returns text tooltip for given mempool depth (in vbytes)."""
+        if depth is None:
+            return "unknown from tip"
+        depth_mb = self.get_depth_mb_str(depth)
+        return _("{} from tip").format(depth_mb)
+
+    @classmethod
+    def get_depth_mb_str(self, depth: int) -> str:
+        # e.g. 500_000 -> "0.50 MB"
+        depth_mb = "{:.2f}".format(depth / 1_000_000)  # maybe .rstrip("0") ?
+        return f"{depth_mb} {util.UI_UNIT_NAME_MEMPOOL_MB}"
+
+    def fee_per_kb(self, network: 'Network') -> Optional[int]:
+        """Returns sat/kvB fee to pay for a txn.
+        Note: might return None.
+        """
+        slider_pos = self.get_slider_pos()
+        if constants.net is constants.BitcoinRegtest:
+            return FEERATE_STATIC_VALUES[slider_pos]
+        if self.method == FeeMethod.MEMPOOL:
+            fee_rate = network.mempool_fees.depth_to_fee(slider_pos)
+        elif self.method == FeeMethod.ETA:
+            fee_rate = network.fee_estimates.eta_to_fee(slider_pos)
+        elif self.method == FeeMethod.STATIC:
+            fee_rate = self.value
+        else:
+            raise Exception(self.method)
+        if fee_rate is not None:
+            fee_rate = int(fee_rate)
+        return fee_rate
+
+    def fee_per_byte(self, network: 'Network') -> Optional[int]:
+        """Returns sat/vB fee to pay for a txn.
+        Note: might return None.
+        """
+        fee_per_kb = self.fee_per_kb(network)
+        return fee_per_kb / 1000 if fee_per_kb is not None else None
+
+    def estimate_fee(self, network: 'Network', size: Union[int, float, Decimal], *,
+                     allow_fallback_to_static_rates: bool = False) -> int:
```

Comment:
Let's make network kwarg-only.
Also, looks like it should be `Optional['Network']`.
```suggestion
    def estimate_fee(
        self,
        size: Union[int, float, Decimal],
        *,
        network: Optional['Network'],
        allow_fallback_to_static_rates: bool = False,
    ) -> int:
```